<launch>

<rosparam command="load" file="$(find reggie_localize)/config/marker_color_ranges.yaml" />

<node name="reggie_localize_node" pkg="reggie_localize" type="reggie_localize_node" output="screen" />

</launch>
